myokit/formats/sbml/_importer.py

```diff
@@ -35,334 +36,1065 @@ def __init__(self):
         self.re_name = re.compile(r'^[a-zA-Z]+[a-zA-Z0-9_]*$')
         self.re_alpha = re.compile(r'[\W]+')
         self.re_white = re.compile(r'[ \t\f\n\r]+')
-        self.units = {}
 
-    def _convert_name(self, name):
+    def info(self):
+        return info
+
+    def supports_model(self):
         """
-        Converts a name to something acceptable to myokit.
+        Returns a bool indicating if model import is supported.
```

Comment:
In PINTS we've started doing (most) docstrings for inherited methods as `"""See :meth:`myokit.format.Importer.supports_model()`."""`.
Think that might be useful here